cves/CVE-2008-2939.yml

```diff
@@ -201,4 +210,8 @@ mistakes:
     Use those questions to inspire your answer. Don't feel obligated to answer
     every one. Write a thoughtful entry here that those ing the software
     engineering industry would find interesting.
-  answer:
+  answer: |
+    The mistake that was made was a coding mistake. The design or developers did
```

Comment:
I think you mean to say the "designers" here.

Reply:
I believe that is the correct way to phrase the mistake. It indeed was a design mistake before the coding phase thank you for the clarification.